II.pass-tests/03-sum-numbers/add-numbers.js

```diff
@@ -1,4 +1,7 @@
-function addNumbers(numbers) {
-
+function addNumbers (numbers) {
+  return numbers.reduce(
```

Comment:
excellent :smiley: 
`reduce` is one of my favourite functions

Reply:
mine too it makes things so easy but it took me time to get my head around it :)